core/dive.c

```diff
@@ -1680,14 +1693,7 @@ static void sample_renumber(struct sample *s, int i, const int mapping[])
 		if (s->sensor[j] != NO_SENSOR)
 			sensor = mapping[s->sensor[j]];
 		if (sensor == -1) {
-			// Remove sensor and gas pressure info
-			if (i == 0) {
-				s->sensor[j] = 0;
-				s->pressure[j].mbar = 0;
-			} else {
-				s->sensor[j] = s[-1].sensor[j];
-				s->pressure[j].mbar = s[-1].pressure[j].mbar;
-			}
+			clear_sensor(s, i, j);
```

Comment:
Ok, so here the new code did what the old code did. Which was always a bit odd, but at least it doesn't change behavior like above.